sw/host/opentitantool/src/backend/ti50.rs

```diff
@@ -0,0 +1,19 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+use anyhow::Result;
+use opentitanlib::transport::ti50::HostEmulation;
+use opentitanlib::transport::Transport;
+use opentitanlib::util::parse_int::ParseInt;
+
+use crate::backend::BackendOpts;
+
+pub fn create(args: &BackendOpts) -> Result<Box<dyn Transport>> {
+    Ok(Box::new(HostEmulation::new(
+        match &args.usb_serial {
+            Some(serial) => Some(u16::from_str(&serial)?),
+            None => None,
+        }
+    )))
```

Comment:
You should add a `port: Option<u16>` to BackendOpts and pass that in here.

Reply:
I am not sure a u16 port is going to be the final way.  It could be a unix socket path, or a hostname:port combination.  I wanted to demonstrate how a parameter could be passed, without messing with BackendOpts before we are sure what we need.